### ISSUE TITLE
Confirmed Cases missing

### ISSUE THREAD

**Issue by JimSorenson** [datasets/covid-19#56]
Current dataset on 4/7/2020 shows 0 cases for North Dakota and the overall # for 4/6/2020 is off by about 140k.

**Comment by kennymalac** [comment]
Indeed, I'm seeing 0 confirmed for both Alaska and Michigan as well
